### PR TITLE
[IMP] mail: follow up canned response

### PR DESCRIPTION
**Pull request by phenix-factory**
This PR removes the "Shared canned response" filter from the default filter and corrects typos on strings.

task-3814167

### CODE REVIEW

File: addons/mail/views/mail_canned_response_views.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
                 <search string="Canned Responses Search">
                     <field name="source"/>
                     <field name="substitution"/>
-                    <filter string="My canned response" name="my_canned_response" domain="[('create_uid', '=', uid)]"/>
-                    <filter string="Shared canned response" name="shared_canned_response" domain="[('is_shared', '=', True)]"/>
+                    <filter string="My canned responses" name="my_canned_response" domain="[('create_uid', '=', uid)]"/>
+                    <filter string="Shared canned responses" name="shared_canned_response" domain="[('is_shared', '=', True)]"/>
```
Comment (tde-banana-odoo) [inline addons/mail/views/mail_canned_response_views.xml:13]:
```suggestion
                    <filter string="My canned responses" name="filter_create_uid" domain="[('create_uid', '=', uid)]"/>
                    <filter string="Shared canned responses" name="filter_is_shared" domain="[('is_shared', '=', True)]"/>
```

**Review by tde-banana-odoo — COMMENTED** [review]
While being there could you use a correct naming for filters ? When possible use filter prefix (eases understanding xpaths) and field naming (as it is more robust).

Cheers !

File: addons/mail/views/mail_canned_response_views.xml (hunk 1 above)
Comment (phenix-factory) [reply addons/mail/views/mail_canned_response_views.xml]: Done

**Comment by tde-banana-odoo** [comment]
As the freeze is about to be done, I suggest to wait then retarget to 17.2 :) .

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Comment by seb-odoo** [comment]
@robodoo r+

**Pull request closed without merging**